src/transformers/hf_argparser.py

```diff
@@ -234,29 +234,65 @@ def parse_args_into_dataclasses(
 
             return (*outputs,)
 
-    def parse_json_file(self, json_file: str) -> Tuple[DataClass, ...]:
+    def parse_json_file(self, json_file: str, allow_extra_keys: bool = True) -> Tuple[DataClass, ...]:
         """
         Alternative helper method that does not use `argparse` at all, instead loading a json file and populating the
         dataclass types.
+
+        Args:
+            json_file:
+                File name of the json file to parse
+            allow_extra_keys:
```

Comment:
```suggestion
            allow_extra_keys (`bool`, *optional*, defaults to `False`):
```
(or `True` if we don't change the default)